src/libraries/Microsoft.VisualBasic.Core/ref/Microsoft.VisualBasic.Core.cs

```diff
@@ -37,20 +39,20 @@ public sealed partial class Collection : System.Collections.ICollection, System.
         bool System.Collections.IList.IsFixedSize { get { throw null; } }
         bool System.Collections.IList.IsReadOnly { get { throw null; } }
         object System.Collections.IList.this[int index] { get { throw null; } set { } }
-        public void Add(object Item, string Key = null, object Before = null, object After = null) { }
+        public void Add(object? Item, string? Key = null, object? Before = null, object? After = null) { }
         public void Clear() { }
         public bool Contains(string Key) { throw null; }
```

Comment:
if you can add a null key, can contains accept a null?

Reply:
Apparently not (unless I'm misreading it):
https://source.dot.net/#Microsoft.VisualBasic.Core/Microsoft/VisualBasic/Collection.vb,150

Reply:
Not familiar with VB but from reading it my logic seems like it won't accept nulls.